### DISCUSSION TITLE
GOD OF WAR 2018 CUSA 07408 v1.34

### DISCUSSION THREAD

**Discussion by czirzone** [GoldHEN/GoldHEN_Cheat_Repository#83] [Cheat requests]
Can someone please show me how to make cheats for version 1.34 cusa 07408? If possible please share with me the cheat of this version. thanks a lot

**Comment by kmeps4** [comment]
Please go to https://github.com/GoldHEN/GoldHEN_Cheat_Repository/discussions/42
